global_config/production.sh

```diff
@@ -2,4 +2,5 @@ CONFIG=production
 var_file=production
 CONFIG_SHORT=pd
 AZURE_SUBSCRIPTION=s189-teacher-services-cloud-production
+AZURE_NAMESPACE=tv-production
```

Comment:
The namespace is set in the terraform configuration json: https://github.com/DFE-Digital/teaching-vacancies/blob/main/terraform/workspace-variables/production.tfvars.json#L44
You could remove duplication by reading from the json using `jq`